jobs/archiver_syslog/templates/bin/archiver_syslog_ctl

```diff
@@ -31,7 +31,7 @@ case $1 in
     # store this processes pid in $PIDFILE, since the exec below doesn't daemonize
     echo $$ > $PIDFILE
 
-    export LS_JAVA_OPTS="-Xms$HEAP_SIZE -Xmx$HEAP_SIZE -DPID=$$"
+    export LS_JAVA_OPTS="-Xms$HEAP_SIZE -Xmx$HEAP_SIZE -DPID=$$ -Djava.io.tmpdir=/var/vcap/sys/temp/archiver_syslog"
```

Comment:
D'oh, was this missing causing all of the trouble?  Should it be set equal to the name passed in when sourcing the script, or keep it specific?

Reply:
Answered my own question after reviewing things with Steve; we're going to give this a shot to see if it works!